Update Alt text for the ESD mitigation page.

acharraggi opened the pull request:
Also tested using a Codespace to make the change. Worked pretty well.

miriamsr commented on docs/source/hardware_and_software_configuration/configuring/managing_esd/managing-esd.rst:
```diff
@@ -38,10 +37,8 @@ produce extremely high electrical currents as the charge that was
-
-   Positively charged robot next to neutral field wall.
+.. image:: images/positivelyChargedRobot.png
+   :alt: Positively charged robot next to neutral field wall.
```
Is there a reason the figure was changed to an image?

miriamsr replied:
The main reason I pointed it out is because we've been discussing using figures over images in the meetings multiple times, so I wanted to make sure that the change was intentional. Same goes for the other cases on this page where figures were replaced with images.

acharraggi replied:
Yes it's been discussed. However, in the proposed style guide (which will be part of the new Contributing section), I mention that not all images require captions, which I think is still true for the Staticide image. I also discuss how to write alt text that is not a copy of the caption. This is done by writing a functional description for screen readers, and an Editorial or Illustrative caption that ties the image to the content or highlights or add details that might not be apparent to the reader.

But there is no surrounding text to provide context for the robot image, so a caption is appropriate.

For the positively charged robot image the functional Alt text is "A positively charged robot touching a neutral field wall." and the caption will be "Example of an ESD event, current will flow from the robot to the field wall." A visual person can see the photo and the caption, the screen reader would read both sentences and I think that is sufficient to fully describe that image and relate it to the What is an ESD event paragraph.

If a caption ends up being a short functional description, it's probably not needed as a caption, a sighted person can see the image.  This is the case where I think the Image directive with Alt text can be used.

I do think the positively charged robot image benefits from having both the alt text AND caption, so thanks for spotting that. Does that work for you? Anything else on this page?

miriamsr replied:
Yes, I understand now. I was just a little confused given all the talk about "using figures over images" but I agree that a caption isn't needed here, so it makes sense that you used an image.

acharraggi commented:
I think this is ready for review again. There was some Slack and Discord discussion ESD shutting down Wifi if a camera was connected to the USB 2.0 port. I've added a warning about using the USB 2.0 port on a few pages.  I also updated the Alt Text for those pages and add a new section Cameras and USB Hubs to the configuring-external-webcam page. This indicates you can connect two webcams to the USB 3 port by using a USB hub.

However, it's no longer a simple one page change. Now 7 pages have been updated, and two new images added.
You can see these pages live at Read the Docs:
https://first-tech-challenge-ftcdocs--340.com.readthedocs.build/en/340/hardware_and_software_configuration/configuring/managing_esd/managing-esd.html

acharraggi commented:
Restored some image centering on some pages since that is how they were initially set up. Did not center images on other pages, if the original page was setup that way.

I wasn't happy with how some images are too big in the PDF, or get stretched. Images that are portrait or images that are small. The PDF formatting process seems to want to expand the image to fill the width, so either they are too big, or get stretched and look pixelated.

Images that are 800 to 1000 pixels wide don't seem to have this problem.
Turns out one way to prevent that is to set the image height to how tall you want the image, then add white space to the image horizontally so that it's about 800 or so pixels wide. This usually means the images are presented about the same size in both HTML and PDF. 

So for example the picture of the Staticide bottles is a portrait image and in the PDF the image fills a page. It doesn't need to be so big. By setting the height to a reasonable size, I expanded the white space around the image to make the width 800 pixels. Now the image looks good in both HTML and PDF.
